src/stories/index.stories.js

```diff
@@ -68,84 +36,20 @@ export const Default = () => {
 
   return (
     <>
-      <CKEditor
-        editor={DecoupledEditor}
-        data={storedValue}
-        config={{
-          plugins: [
-            Essentials,
-            Bold,
-            Italic,
-            Paragraph,
-            Underline,
-            Code,
-            HorizontalLine,
-            Link,
-            Indent,
-            IndentBlock,
-            List,
-            BlockQuote,
-            Heading,
-            CodeBlockEditing,
-            CodeBlockUI,
-            Widget
-          ],
-          toolbar: [
-            'heading',
-            'bold',
-            'italic',
-            'underline',
-            'code',
-            '|',
-            'bulletedList',
-            'numberedList',
-            'outdent',
-            'indent',
-            '|',
-            'blockquote',
-            'codeBlock',
-            'link',
-            'horizontalLine'
-          ],
-          codeBlock: {
-            languages: [{ language: 'plaintext', label: 'Plain text' }]
-          }
-        }}
-        onReady={editor => {
-          /**
-           * Apply custom keyboard shortcuts
-           */
-          editor.keystrokes.set('CTRL+SHIFT+8', 'bulletedList');
-          editor.keystrokes.set('CTRL+SHIFT+7', 'numberedList');
-          editor.keystrokes.set(['ctrl', 219], 'outdent'); // CTRL+[
-          editor.keystrokes.set(['ctrl', 221], 'indent'); // CTRL+]
-          editor.keystrokes.set('CTRL+SHIFT+9', 'blockQuote');
-          editor.keystrokes.set('CTRL+SHIFT+5', 'code');
-          editor.keystrokes.set('CTRL+SHIFT+L', 'horizontalLine');
-
-          /**
-           * Attach decoupled toolbar
-           */
-          const toolbarContainer = document.querySelector('.toolbar-container');
-
-          toolbarContainer.appendChild(editor.ui.view.toolbar.element);
-
-          /**
-           * Disable spell checker for demo content
-           */
-          editor.editing.view.change(writer => {
-            writer.setAttribute('spellcheck', 'false', editor.editing.view.document.getRoot());
-          });
-        }}
+      <Editor
+        data={data}
         onChange={(event, editor) => {
           const editorData = editor.getData();
 
           action('onChange')(editorData);
 
           onValueChange(editorData);
         }}
+        toolbarClassName={TOOLBAR_CLASS_NAME}
       />
-      <div className="toolbar-container" />
+      <div className="toolbar-container">
+        <div className={`${TOOLBAR_CLASS_NAME}`} />
```

Comment:
Could this be simplified to `<div className={TOOLBAR_CLASS_NAME} />` since the JS template expression only includes the string?

Reply:
Of course 😅